website/docs/guides/mesh-qs.md

```diff
@@ -0,0 +1,563 @@
+---
+title: "Quickstart for dbt Mesh"
+id: "mesh-qs"
+level: 'Intermediate'
+icon: 'guides'
+tags: ['dbt Cloud','Quickstart']
```

Comment:
replaced intermediate w quickstart here since we have intermediate above. otherwise intermediate was showing up twice in the doc.

Reply:
sounds good!